website/blog/2025-06-09-moving-towards-a-stable-javascript-api.md

```diff
@@ -0,0 +1,322 @@
+
+The timeline for this will be based on community feedback. For at least the next two React Native releases, the Strict API will remain an opt-in.
+
+## FAQs
+
+<details>
+<summary>
+**I'm using subpath imports today. What should I do?**
+</summary>
+
+Please migrate to the root `'react-native'` import path.
+
+- Subpath imports (e.g. `'react-native/Libraries/Alert/Alert'`) are becoming private APIs. Without preventing access to implementation files inside React Native, we can’t offer a stable JavaScript API.
+- We want our deprecation warnings to motivate community feedback, which can be raised via our [centralized discussion thread](https://github.com/react-native-community/discussions-and-proposals/discussions/893), if you believe we are not exposing code paths that are crucial for your app. Where justified, we may promote APIs to the index export.
+
+</details>
+
+<details>
+<summary>
+**I'm a library maintainer. How does this change impact me?**
+</summary>
+
+Both apps and libraries can opt in at their own pace, since `tsconfig.json` will only affect the immediate codebase.
+
+- Typically, `node_modules` is excluded from validation by the TypeScript server in a React Native project. Therefore, your package's exported type definitions are the source of truth.
+
+**💡 We want feedback!** As with changed subpath imports, if you encounter any integration issues with the Strict API, please let us know [on GitHub](https://github.com/react-native-community/discussions-and-proposals/discussions/893).
+
+</details>
+
+<details>
+<summary>
+**Does this guarantee a final API for React Native yet?**
+</summary>
+
+Sadly, not yet. In 0.80, we've made a tooling investment so that React Native's existing JS API baseline can be accurately consumed via TypeScript — enabling future stable changes.
+
+Later in 2025 (and beyond), we may take more action to finalise the APIs we currently offer in core. API changes will always be communicated via RFCs/announcements, and typically a deprecation cycle.
+
+- It's worth noting that React Native's JavaScript API **has already been pretty stable** across past `0.x` major versions. We're formalizing the existing API you know and love.
+
+</details>
+
+<details>
+<summary>
+**Why isn't React Native written in TypeScript?**
+</summary>
+
+React Native is core infrastructure at Meta. We test every merged change across our Family of Apps, before they hit general open source availability.
+
+At this scale and sensitivity, correctness matters. The bottom line is that Flow offers us greater performance and greater strictness than TypeScript, including specific [multi-platform support for React Native](https://flow.org/en/docs/react/multiplatform/).
+
+</details>
+
+## Thanks
+
+These changes were made possible by [Alex Hunt](https://x.com/huntie), [Riccardo Cipolleschi](https://x.com/CipolleschiR), [Iwo Plaza](https://x.com/iwoplaza), [Jakub Piasecki](https://x.com/breskin67), and [Dawid Małecki](https://github.com/coado).
```

Comment:
I would consider either: 
- sort them alphabetically
_or_
- put SWM people before Meta people because they did most of the work and because I feel it is more polite ("guests" first)